### PR TITLE
Repl.it presence

### PR DESCRIPTION
**Pull request by Jakeisbored**

### CODE REVIEW

**Comment by Jakeisbored** [comment]
image
image
image

File: Repl.it/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+        "id": "480407581085532180"
+    },
+    "service": "Repl.it",
+    "description": {
+        "en": "Repl.it is a San Francisco based start-up and an online IDE (integrated development environment) supporting 40 programming languages, including Python, Lua, JavaScript, Julia, C++, C++11, C, C#, HTML/CSS/JS, Python with turtle and LOLCODE. Repl.it allows users to write code and build apps using a browser, without having to install any software on their devices."
+    },
+    "url": "repl.it",
+    "version": "1.0",
+    "logo": "https://upload.wikimedia.org/wikipedia/commons/thumb/b/b2/Repl.it_logo.svg/512px-Repl.it_logo.svg.png",
+    "thumbnail": "https://pasteboard.co/IXzehgY.png",
```
Comment (Bas950) [inline Repl.it/dist/metadata.json]: upload to imgur.

File: Repl.it/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    "color": "grey",
+    "tags": [
```
Comment (Bas950) [inline Repl.it/dist/metadata.json]: use hex color codes, for the color

File: Repl.it/dist/metadata.json (hunk 1 above)
Comment (Jakeisbored) [reply Repl.it/dist/metadata.json]: should the logo also be on imgur?

File: Repl.it/dist/metadata.json [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+        "id": "480407581085532180"
+    },
+    "service": "Repl.it",
+    "description": {
+        "en": "Repl.it is a San Francisco based start-up and an online IDE (integrated development environment) supporting 40 programming languages, including Python, Lua, JavaScript, Julia, C++, C++11, C, C#, HTML/CSS/JS, Python with turtle and LOLCODE. Repl.it allows users to write code and build apps using a browser, without having to install any software on their devices."
+    },
+    "url": "repl.it",
+    "version": "1.0",
+    "logo": "https://imgur.com/a/QPWqO6f",
+    "thumbnail": "https://imgur.com/a/pz3PEbc",
```
Comment (Bas950) [inline Repl.it/dist/metadata.json]: should be i.imgur links ending with .png .jpg etc.

**Pull request merged**